corehq/apps/locations/fixtures.py

```diff
@@ -265,92 +258,6 @@ def _cte_get_location_fixture_queryset(user):
     return result
 
 
-def _mptt_get_location_fixture_queryset(user):
```

Comment:
At some point I think I'd like to explore reintroducing a variant of this that's compatible with the cte approach to see how it compares against `get_location_fixture_ids.sql`.  As I recall, the blocker was the use of `|` to join query components, right?  Wasn't there some workaround for that?

Reply:
There's not an easy workaround for that that I know of, but I'd be happy to answer any questions about it if you want to give it another try.